src/main/java/org/opendevstack/provision/services/JenkinsPipelineAdapter.java

```diff
@@ -416,6 +413,16 @@ private Execution buildExecutionObject(
     return execution;
   }
 
+  private String computeWebhookProxyHost(String jobId, String projID) {
```

Comment:
are these 3 cases in this method unit tested?

Reply:
As the change only modifies the existing functionality the existing integration / e2e tests were adapted and test the functionality.
See the changes in:
```
org.opendevstack.provision.controller.E2EProjectAPIControllerTest#testProvisionNewSimpleProjectInternal
org.opendevstack.provision.controller.E2EProjectAPIControllerTest#testQuickstarterProvisionOnNewOpenProjectInclDeleteWholeProject
org.opendevstack.provision.controller.E2EProjectAPIControllerTest#testQuickstarterProvisionOnNewOpenProjectInclDeleteSingleComponent
```